Facepunch.Steamworks/SteamUtils.cs

```diff
@@ -261,5 +261,20 @@ internal static bool IsCallComplete( SteamAPICall_t call, out bool failed )
 		/// Returns whether this steam client is a Steam China specific client, vs the global client
 		/// </summary>
 		public static bool IsSteamChinaLauncher => Internal.IsSteamChinaLauncher();
+
+		/// <summary>
+		/// Initializes text filtering, loading dictionaries for the language the game is running in.
+		/// Users can customize the text filter behavior in their Steam Account preferences
+		/// </summary>
+		public static bool InitFilterText( uint unFilterOptions ) => Internal.InitFilterText( unFilterOptions );
+
+		/// <summary>
+		/// Filters the provided input message and places the filtered result into pchOutFilteredText,
+		/// using legally required filtering and additional filtering based on the context and user settings.
+		/// </summary>
+		public int FilterText( TextFilteringContext eContext, SteamId sourceSteamID, string pchInputMessage, out string pchOutFilteredText )
```

Comment:
could this be simpler? string in and string out? why do we return an int?

Reply:
I can do this

Reply:
The integer returned is the number of characters that is filtered, I think in the most common usages it will not be used.